celery/app/defaults.py

```diff
@@ -291,6 +291,7 @@ def __repr__(self):
         ),
         store_errors_even_if_ignored=Option(False, type='bool'),
         track_started=Option(False, type='bool'),
+        allow_error_cb_on_chord_header=Option(False, type='bool'),
```

Comment:
we should document this part properly

Reply:
Please review the PR again @auvipy , I've added proper documentation